Gopkg.toml

```diff
@@ -32,7 +32,7 @@
 
 [[constraint]]
   name = "github.com/topfreegames/go-gcm"
-  version = "1.3.0"
+  branch = "master"
```

Comment:
We should make a release in go-gcm and use it here. I'm not keen on referencing "master".